content/guides/tasks/assignments/update-message.md

```diff
@@ -0,0 +1,75 @@
+---
+rank: 4
+related_endpoints:
+  - put_task_assignments_id
+related_guides: []
+required_guides: []
+alias_paths: []
+---
+
+# Change a task assignment message
+
+To update a task assignment in Box you will need to call the
+[`PUT /tasks/:task_id/assignments`](e://put_task_assignments_id) API, and
+include the new `message` for the task assignment.
+
+<Tabs>
+  <Tab title="cURL">
+    <!-- markdownlint-disable line-length -->
```

Comment:
For future ref just feel free to wrap the whole tabs section in this markdown-disable line.

Reply:
Yeah, I copied something I saw you do and thought I could do that but mostly wanted to get it done. Might add that to internal docs

Reply:
All the SDK samples were merged, so I've updated this to use variants